src/components/image/image.tsx

```diff
@@ -238,11 +257,12 @@ export const Image = component$<ImageProps>((props) => {
 		<img
 			decoding="async"
 			{...imageAttributesWithoutChildren}
+			src={srcWithPreset.value}
```

Comment:
The original implementation was using `srcset` to change the image after the download. 🤔
I didn't the the solution but it looks like a conflict with the actual behaviour.